internal/pkg/flink/components/.snapshots/TestTableViewTestSuite-TestTableInfoBarWhenTableTableHasNoContentAndAutoRefreshIsOff

```diff
@@ -0,0 +1,5 @@
+([]string) (len=3) {
+  (string) (len=19) "Fetch state: paused",
```

Comment:
Two things about naming:

Fetch state -> Refresh
paused/running -> Paused / Running

Refresh is the same naming they use in the open search version and I think it's just better and shorter than fetch state.

Paused and Running look a bit weird in lower case, because we always have the first letter in upper case. So I would either write the first letter or the whole word in upper case

Reply:
Changed all states to be uppercase and the prefix from "Fetch state:" to "Refresh:"!